### ISSUE TITLE
DatePickerDialog not displaying in iOS13

### ISSUE THREAD

**Issue by ibhavin** [squimer/DatePickerDialog-iOS-Swift#101]
DatePickerDialog-iOS-Swift is working in all previous iOS version but in iOS 13 Dialog is not displaying.

**Comment by ezemartinezna** [comment]
Same here... :(

**Comment by ibhavin** [comment]
From Pull Request:

Replace this code:
```
/* Add dialog to main window */
guard let appDelegate = UIApplication.shared.delegate else { fatalError() }
guard let window = appDelegate.window else { fatalError() }
window?.addSubview(self)
window?.bringSubviewToFront(self)
window?.endEditing(true)
```

To this code:
```
/* Add dialog to main window */
if #available(iOS 13.0, *) {
    if let currentWindowScene = UIApplication.shared.connectedScenes.first as? UIWindowScene {
        guard let window = currentWindowScene.windows.first else { fatalError() }
        window.addSubview(self)
        window.bringSubviewToFront(self)
        window.endEditing(true)
    }
} else {
    guard let appDelegate = UIApplication.shared.delegate else { fatalError() }
    guard let window = appDelegate.window else { fatalError() }
    window?.addSubview(self)
    window?.bringSubviewToFront(self)
    window?.endEditing(true)
}
```

It will work on iOS 13 too.
